### PR TITLE
Vaishnavi_Assignment_31_25min_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image

### CODE REVIEW

**Review by harshvegada — APPROVED** [review]
Well handled scenarios when user not expected input
Good u used array but this will done without using array

File: src/vaishnaviVJun20_ExceptionHandling/SumOfFirstThreeNumbers.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.Scanner;
+
+public class SumOfFirstThreeNumbers {
+	Scanner scanner = new Scanner(System.in);
+	int count = 0, sum = 0;
+	int[] array = new int[3];
+
+	void getSumOfFirstThreeNumbers() {
+		System.out.println("Please enter numbers to add: ");
+		do {
```
Comment (harshvegada) [inline src/vaishnaviVJun20_ExceptionHandling/SumOfFirstThreeNumbers.java:26]: Good u used do while loop
Different than other's

**Comment by VaishnaviVJun20** [comment]
> Well handled scenarios when user not expected input
> Good u used array but this will done without using array

-- Thank you @harshvegada.
--Yes, I tried without array first and it was handling the program correctly, However I wasn't able to achieve suggested output as we also  need to print which numbers were entered as input by user without using array.
-- Please suggest if there is any way to do that, I would like to practice it as well :)

File: src/vaishnaviVJun20_ExceptionHandling/SumOfFirstThreeNumbers.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20_ExceptionHandling/SumOfFirstThreeNumbers.java:26]: Thank you so much @harshvegada :)

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**